```
==> make install DESTDIR=/home/linuxbrew/.linuxbrew/Cellar/govuk-cli/0.0.1
Last 15 lines from /home/isabelllong/.cache/Homebrew/Logs/govuk-cli/01.make:
2019-09-10 09:25:14 +0100

make
install
DESTDIR=/home/linuxbrew/.linuxbrew/Cellar/govuk-cli/0.0.1

make: *** No rule to make target 'install'.  Stop.
```

--- a/Formula/govuk-cli.rb
+++ b/Formula/govuk-cli.rb
@@ -3,7 +3,7 @@ class GovukCli < Formula
   homepage "https://github.com/alphagov/govuk-cli.git"
   # The repo doesn't tag versions (yet).
   url "https://github.com/alphagov/govuk-cli.git"
-  version "0.0.1"
+  version "0.0.2"
   head "https://github.com/alphagov/govuk-cli.git"
 
   def install

